_includes/head.html

```diff
@@ -43,6 +43,12 @@
     type="text/css"
   />
 
+
+
+</script>
+<!--adds the analytics.html script to every page-->
+{% include analytics.html %}
```

Comment:
Will this `{% include analytics.html %}` be executed or is it just a comment. Because if it is a comment I didn;t get how `analytics.html` is included.

Reply:
{% include analytics.html %} is not a comment. It is a command to be executed to ensure analytics.html is included on all pages.